upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -1184,7 +1184,7 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		}
 		if b.cluster.Spec.Authentication.Aws != nil {
 			key := "authentication.aws"
-			version := "0.3.0-kops.1"
+			version := "0.4.0"
```

Comment:
Do we want to keep the “-kops.1” part of the version here? We normally add this in case we find an issue with the spec and want to revision it while still keeping with the upstream versioning.

Reply:
I wasn't sure. I saw other versions in that file that dont include the kops suffix and I had added it when adding support for overriding the image altogether a few weeks ago. I'm happy to keep the suffix if you think it's beneficial.